figures/permissions.py

```diff
@@ -41,23 +44,21 @@ def is_site_admin_user(request):
     3. Get the user org mappings for the orgs and user in the request
     4. Check the uom record if user is admin and active
     """
-    has_permission = is_active_staff_or_superuser(request)
-    if not has_permission:
-        if figures.helpers.is_multisite():
-            if request.user.is_active:
-                current_site = django.contrib.sites.shortcuts.get_current_site(request)
-                org_ids = Organization.objects.filter(
-                    sites__in=[current_site]).values_list('id',
-                                                          flat=True)
-                return UserOrganizationMapping.objects.filter(
-                    organization_id__in=org_ids,
-                    user=request.user,
-                    is_active=True,
-                    is_amc_admin=True).exists()
-            else:
-                return False
+    if figures.helpers.is_multisite():
+        if request.user.is_active:
+            current_site = django.contrib.sites.shortcuts.get_current_site(request)
+            has_permission = is_active_staff_or_superuser(request) or EdlyUserProfile.objects.filter(
+                edly_sub_organizations__lms_site=current_site,
+                user=request.user,
+                user__groups__name__in=[
+                    settings.EDLY_PANEL_ADMIN_USERS_GROUP,
```

Comment:
Also add unittests that both `EDLY_PANEL_ADMIN_USERS` and `EDLY_PANEL_USERS` can access Figures API's  a user which is not in this group cannot access Figures API's  .

Reply:
unittests are already updated for checking this by creating a user with panel access. (https://github.com/edly-io/figures/blob/0fefdff919d86baaa1963cd3ddacca2c3387cc50/tests/test_permissions.py#L97)